### PR TITLE
Portfolio Project: Local storage

### PR DESCRIPTION
**Pull request by Devtiwo**
In this project, i 

- Added id to the form tags
- Made changes to the style.css file
- Implemented local storage functionality

### CODE REVIEW

**Review by enaburekhan — CHANGES_REQUESTED** [review]
Hi Devtiwo and Pydev, 

Good job so far!

- [ ] Major highlights

- PR summary and commit messages are very descriptive
- The codebase is very clean, easy to read and understand
- Passed linters checks

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ] Your readme looks very professional. However, we can make it more descriptive by adding how to clone the app and run it locally. This can be achieved in the getting started section. So I highly recommend you add these steps to the getting started section in your readme.

I had to add this comment here because the readme was not included in the in-line comments.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Code review by **Eric Enaburekhan**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: app.js [hunk 1]
```diff
@@ -161,4 +161,32 @@ form.addEventListener('submit', (e) => {
...
+
+formInput.forEach((fi) => {
+  fi.addEventListener('input', () => {
+    const storeObj = {
+      name: inname.value,
+      email: inemail.value,
+      text: text.value,
+    };
+    setlocalstorage('formData', storeObj);
+  });
```
Comment (brytebee) [inline app.js:191]: - [ ] **Required** In this project, we are required to collect data from the form and store them in the local storage and display them if/when the page reloads without submitting the form. Right now it is not doing this.
**HINT** First store the data in the local storage `localStorage.setItem` on firing an event of your choice that listens for the input. Then display this data when page reload if it exists in the local storage `localStogae.getItem`. I hope this makes sense 😏.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Devtiwo 👋, 

Great work on making the changes 👏 👏

## Highlight 💡💡

- ✔️ Good usage command
- ✔️ Clean code

You've done well implementing some of the requested changes requested by the previous review, but there are still some which aren't addressed yet.


# Suggested changes
_Check the comments under the review._




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by t-yanick — APPROVED** [review]
# STATUS: APPROVED 👍  🥇 👏 
### Hi @Devtiwo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**